### PR TITLE
(ruka/loki) user creation for loki to access bucket on ceph

### PR DESCRIPTION
**Pull request by gseriche**

### CODE REVIEW

File: fleet/lib/rook-ceph-conf/charts/ruka/templates/cephobjectstoreuser-loki.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    - kind: ClusterSecretStore
+      name: onepassword-oods
+  selector:
+    secret:
+      name: rook-ceph-object-user-lfa-loki
+  data:
+    - match:
+        secretKey: AccessKey
+        remoteRef:
+          remoteKey: loki
```
Comment (dtapiacl) [inline fleet/lib/rook-ceph-conf/charts/ruka/templates/cephobjectstoreuser-loki.yaml:27]: I would rename this to `loki-dev` to refer to the site this is going to.

File: fleet/lib/rook-ceph-conf/charts/ruka/templates/cephobjectstoreuser-loki.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+---
+apiVersion: external-secrets.io/v1alpha1
+kind: PushSecret
+metadata:
+  name: rook-ceph-object-user-lfa-loki
+  namespace: rook-ceph
+spec:
+  secretStoreRefs:
+    - kind: ClusterSecretStore
+      name: onepassword-oods
```
Comment (dtapiacl) [inline fleet/lib/rook-ceph-conf/charts/ruka/templates/cephobjectstoreuser-loki.yaml:19]: Is this the correct place where these credentials will be stored?

File: fleet/lib/rook-ceph-conf/charts/ruka/templates/cephobjectstoreuser-loki.yaml (hunk 1 above)
Comment (gseriche) [reply fleet/lib/rook-ceph-conf/charts/ruka/templates/cephobjectstoreuser-loki.yaml:27]: Ok, you're right.

File: fleet/lib/rook-ceph-conf/charts/ruka/templates/cephobjectstoreuser-loki.yaml (hunk 2 above)
Comment (dtapiacl) [reply fleet/lib/rook-ceph-conf/charts/ruka/templates/cephobjectstoreuser-loki.yaml:19]: @jhoblitt  Could probably best answer that.

**Review by jhoblitt — CHANGES_REQUESTED** [review]
The `CephObjectStoerUser` CR looks good.

The PR title and commit messages do not need to include the Jira ticket slug (IT-5699).  The slug being part of the branch name is sufficient for linking the branch to the Jira issue.

The commits should be squashed into a single changeset before this PR is merged.  The general goal is to keep the git history useful. 5 commits to introduce a new resource may be confusing to someone reviewing the repo history.

**Review by jhoblitt — CHANGES_REQUESTED** [review]
Please squash down to a single commit:
image

**Review by jhoblitt — CHANGES_REQUESTED** [review]
https://www.youtube.com/watch?v=sqcLjcSloXs ... "commit"

Also, there are now two unnecessary empty files in this PR:

image

**Pull request closed without merging**